doc/users/next_whats_new/stripey_lines.rst

```diff
@@ -0,0 +1,19 @@
+Stripey Lines
```

Comment:
"Stripey" feels more natural to me but I don't have a strong opinion.  Since the original PR and the issue used "striped" that's at least 3 votes to 1!